instagram/src/components/Login/Login.js

```diff
@@ -25,13 +26,15 @@ class Login extends Component {
         return (  
             <form>
                 <input
+                name="username"
                 type="text" 
```

Comment:
`name` here on this line links to `e.target.name` in your `setState` declaration.